.github/workflows/python-tox.yml

```
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```
We probably don't need 3.8 any longer

---

I wasn't sure about this. On one hand, python 3.8 has reached end-of-life. On the other, it is listed as compatible with Wagtail 5.2. My instinct was to wait until 5.2 becomes deprecated to remove it.

---

Dropped 3.8 in a6856347431eab0e2e61616b54045a75b2a7bf89